view/partial/result.html

```diff
@@ -116,8 +116,8 @@ <h4 class="crunch-top">View results in browser</h4>
 
 							<div class="task-details collapse" id="error-index-{{@index}}">
 								{{#if solutions.length}}
-									<div class="subtitle" id="error-solutions">Solutions:</div>
-									<ul class="list-unstyled solutions-list" role="list" aria-labelledby="error-solutions">
+									<div class="subtitle">Solutions:</div>
+									<ul class="list-unstyled solutions-list" role="list">
```

Comment:
```suggestion
									<ul class="list-unstyled solutions-list">
```

`role="list"` is unnecessary on list elements (`ul` and `ol` have existed since version 1 of HTML, they don't need aria support to be understandable by assistive tech)